src/scripts/main.js

```diff
@@ -1,3 +1,349 @@
+            board[i].textContent = '';
+
+            currentScore = parseInt(board[currentIndex + 1].textContent, 10);
+
+            score.textContent = (parseInt(score.textContent, 10)
+              + currentScore).toString();
+
+            moved = true;
+          } else if (currentIndex !== i) {
+            board[currentIndex].classList = [`field-cell field-cell--${board[i].textContent}`];
+            board[currentIndex].textContent = board[i].textContent;
+            board[i].classList = ['field-cell'];
+            board[i].textContent = '';
+
+            moved = true;
+          }
+        }
+      }
+      break;
+    case 'left':
+      for (let i = 1; i <= 15; i++) {
+        if (board[i].classList.length > 1) {
+          let currentIndex = i;
+
+          while (
+            currentIndex > 0
+            && currentIndex !== 4
+            && currentIndex !== 8
+            && currentIndex !== 12
+            && board[currentIndex - 1].textContent === ''
+          ) {
+            currentIndex--;
+          }
+
+          if (
+            currentIndex > 0
+            && currentIndex !== 4
+            && currentIndex !== 8
+            && currentIndex !== 12
+            && board[currentIndex - 1].textContent === board[i].textContent
+          ) {
+            board[currentIndex - 1].textContent *= 2;
+            board[currentIndex - 1].classList = [`field-cell field-cell--${board[currentIndex - 1].textContent}`];
+
+            board[i].classList = ['field-cell'];
+            board[i].textContent = '';
+
+            currentScore = parseInt(board[currentIndex - 1].textContent, 10);
+
+            score.textContent = (parseInt(score.textContent, 10)
+              + currentScore).toString();
+
+            moved = true;
+          } else if (currentIndex !== i) {
+            board[currentIndex].classList = [`field-cell field-cell--${board[i].textContent}`];
+            board[currentIndex].textContent = board[i].textContent;
+            board[i].classList = ['field-cell'];
+            board[i].textContent = '';
+
+            moved = true;
+          }
+        }
+      }
+      break;
+  }
+
+  if (moved) {
+    addRandomCell();
+    checkingSituations(moved);
+  }
+};
+
+function checkingSituations() {
+  const checkBoard = [...board];
+
+  checkBoard.forEach(cell => {
+    if (cell.textContent === '2048') {
+      messageWin.classList.remove('hidden');
+    }
+  });
+
+  const checkLoss = checkBoard.filter(cell => cell.classList.length > 1);
+
+  if (checkLoss.length === 16) {
+    if (!hasAdjacentEqualValue()) {
+      messageLose.classList.remove('hidden');
+      document.removeEventListener('keydown', arrow => arrowTracking(arrow));
+    }
+  }
+}
+
+function hasAdjacentEqualValue() {
+  for (let index = 0; index < board.length; index++) {
+    const row = Math.floor(index / 4);
+    const col = index % 4;
+
+    const neighbors = [
+      { row: row, col: col - 1 }, // слева
```

Comment:
Remove comments everywhere
```suggestion
      { row: row, col: col - 1 }
```